src/assets/persons.js

```diff
@@ -15465,5 +15465,21 @@ export const data = {
         country: "Brazil",
       },
     },
+    ,{
```

Comment:
This extra comma is causing the build failure

Reply:
Thanks for your feedback. I have now removed the extra comma.